docs/tutorials/07-batch-infer.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
```

Comment:
<p>If you do perform drift analysis, a new model endpoint record is generated. Model endpoint is a unique MLRun entity that includes statistics and important details about your model and function. You can perform the drift analysis on an existing model endpoint but you need to make sure that you don't mix unrelated datasets that could affect the final drift analysis process. In general, it's recommended to perform the drift analysis on a new model endpoint to avoid possible analysis conflicts.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/4273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>